addons/website_blog/tests/test_performance.py

```diff
@@ -38,11 +44,11 @@ def setUp(self):
             "cover_properties": """{"background-image": "url('/website_blog/static/src/img/cover_1.jpg')", "resize_class": "o_record_has_cover o_half_screen_height", "opacity": "0"}""",
         } for blog in blogs])
 
-    def test_10_perf_sql_blog_standard_data(self):
+    def test_100_perf_sql_blog_standard_data(self):
```

Comment:
TODO: once runbot enterprise build is green, restore the name here (since the test is disabled runbot wise, can't be tested otherwise)